tests/distributions/test_multivariate.py

```diff
@@ -835,6 +835,23 @@ def test_car_matrix_check(sparse):
             car_dist = pm.CAR.dist(mu, W, alpha, tau)
 
 
+@pytest.mark.parametrize("alpha", [1, -1])
+def test_car_alpha(alpha):
+    """
+    Tests the check that -1 < alpha < 1
+    """
+
+    W = np.array([[0, 1, 0], [1, 0, 1], [0, 1, 0]])
+
+    tau = 1
+    mu = np.array([0, 0, 0])
+
+    car_dist = pm.CAR.dist(W=W, alpha=alpha, mu=mu, tau=tau)
+
+    with pytest.raises(ValueError, match="the domain of alpha is: -1 < alpha < 1"):
+        pm.draw(car_dist)
```

Comment:
Maybe worth to test the logp as well?

Reply:
Yeah I think so too. Is it okay if I write it within the same test or do we need two separate ones?

Reply:
I think either is OK, it's clear what you're testing either way